app/utilities/schema.py

```diff
@@ -27,34 +29,32 @@
 }
 
 
-def get_schema_path_map_for_language(language_code):
-    schema_files = []
-
-    for schema_dir in DEFAULT_SCHEMA_DIRS:
-        schema_files.extend(glob(f"{schema_dir}/{language_code}/*.json"))
-
-    return {
-        Path(schema_file).with_suffix("").name: schema_file
-        for schema_file in schema_files
-    }
+@lru_cache(maxsize=None)
+def get_schema_list(language_code: str = DEFAULT_LANGUAGE_CODE) -> List:
+    return list(get_schema_path_map(include_test_schemas=True)[language_code].keys())
 
 
-def get_schema_path_map():
-    language_map_codes = ["en", "cy", "ga", "eo"]
+@lru_cache(maxsize=None)
+def get_schema_path_map(include_test_schemas: Optional[bool] = False) -> Mapping:
+    dirs = [SCHEMA_DIR]
+    if include_test_schemas:
+        dirs.append(TEST_SCHEMA_DIR)
 
     return {
-        language_code: get_schema_path_map_for_language(language_code)
-        for language_code in language_map_codes
+        language_code: {
+            Path(schema_file).with_suffix("").name: schema_file
+            for schema_dir in dirs
+            for schema_file in glob(f"{schema_dir}/{language_code}/*.json")
+        }
+        for language_code in {"en", "cy", "ga", "eo"}
```

Comment:
I think these language codes should be defined in a constant.

Reply:
Module level?